.vscode/launch.json

```diff
@@ -36,7 +36,7 @@
             "stopOnEntry": false,
             "sourceMaps": true,
             "outFiles": [
-                "${workspaceFolder}/dist/**/*.js"
+                "${workspaceFolder}/out/**/*.js"
```

Comment:
Another way to fix this would be to let the debugger just search the entire workspace, this would be done by simply removing `outFiles`. This is what I did to be able to hit breakpoints. As long as it works I'm okay with just switching the path, but I thought I'd mention it.

Reply:
Thanks for mentioning it, indeed that might be more clear. I only followed the example in the .vscode/launch.json of cpptools and cmake tools extensions. I wonder if there are any consequences from removing "outFiles"... I'll keep this in mind for an eventual later update